### DISCUSSION TITLE
Error: AuthRetryableFetchError, status:504

### DISCUSSION THREAD

**Discussion by CMBurnett** [supabase/supabase#29956] [Questions]
I keep getting an `AuthRetryableFetchError,status:504` error on my NextJS project and am stumped. I get the logs in Supabase and it tells me `error: context deadline exceeded` so it appears it's making it to Supabase. I'd added a retry and tried to change the timeout - no go.

I have the same implementation in another project and it works fine.

Anybody have any ideas or good insight?

**Comment by GaryAustin1** [comment]
Any errors in the auth or Postgres logs give more info?

What are you doing when you get it?

**Reply by GaryAustin1** [reply]
No auth hooks, no triggers on auth.users?
Maybe play with the SMTP provider and see if there is a delay from that (although that is usually called out in the auth error a little better).

**Reply by CMBurnett** [reply]
No hooks, triggers or functions.

I did set up the custom SMTP - maybe there's an issue with that, I'll look. But that shouldn't cause a timeout and the user not being added to the table, yea?

**Reply by CMBurnett** [reply]
Conversely - I'm really early in this project, so I could probably kill this supa project and start fresh to see if that works.

**Reply by GaryAustin1** [reply]
I've seen cases of users with extremely slow SMTP providers.  One over 5 seconds or so and another with a timeout, but the error was different and on sending and email.

**Comment by CMBurnett** [comment]
Nothing on the Postgres logs I can see. But the Auth logs error I posted above: `error: context deadline exceeded`

Interestingly I tried to `Invite` a User from my Supabase Auth Dashboard and it timed out as well. No clue what's going on.

![Screenshot 2024-10-17 at 2 12 32 PM](https://github.com/user-attachments/assets/e1fe079b-70f9-4a24-95c2-6464f96c598a)
